seed/challenges/00-getting-started/getting-started.json

```diff
@@ -1274,6 +1274,17 @@
             ]
           ]
         },
+        "it": {
+          "title": "Leggi notizie sul mondo della programmazione nelle nostre pubblicazioni su Medium",
+          "description": [
+            [
+              "//i.imgur.com/AZNLvEC.gif",
+              "Una gif mostrando come puoi cliccare sul tasto \"follow\" per seguire le pubblicazioni di Free Code Camp.",
+              "La nostra comunità ha uno spazio su Medium dove pubblica numerosi articoli pratici parlando di programmazione e di tecnologia. Una volta aperto il link, clicca sul tasto \"follow\" in alto a destra. Se non hai un account di Medium, puoi crearne uno in pochi secondi.",
```

Comment:
"...~~parlando~~ **che parlano**..." same as before. In Italian the "gerundio" verbal form is not used this way.